examples/api/package.json

```diff
@@ -12,24 +12,25 @@
     "build": "run-s copy-license sync-package-info build:*",
     "build:typescript": "tsc -b ./src/tsconfig.json",
     "watch": "npm-run-all copy-license sync-package-info --parallel watch:*",
-    "watch:server": "nodemon --inspect=0.0.0.0:9240 ./dist/index.js",
+    "watch:server": "nodemon --exec 'node -r dotenv/config' --inspect=0.0.0.0:9240 ./dist/index.js",
```

Comment:
`-r` option is directly supported by `nodemon` e.g.: `nodemon -r dotenv/config`. But `dotenv/config` should probably be loaded directly in `index.ts`